improver/between_thresholds.py

```diff
@@ -82,22 +86,30 @@ def _slice_cube(self):
                 If any of the required constraints returns None
         """
         thresh_coord = self.cube.coord(self.thresh_coord.name())
-        error_string = (thresh_coord.name() + ' threshold {} ' +
-                        self.threshold_units + ' is not available\n')
-        error_msg = ''
+        error_string = (
+            thresh_coord.name()
+            + " threshold {} "
+            + self.threshold_units
+            + " is not available\n"
+        )
```

Comment:
After the PR this can probably be rewritten

Reply:
Gotta say, this layout is a lot less readable than the original.  Not a battle I'm going to win but I don't think this is an improvement!